build-scripts/runtime_lib.cmake

```diff
@@ -1,6 +1,46 @@
 # Copyright (C) 2019 Intel Corporation. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+file(WRITE "${PROJECT_BINARY_DIR}/try_compile_clock_nanosleep.c"
+"   #include <time.h>
+    void mysleep_ms(int milisec)
```

Comment:
Is it possible that the code compiles but it fails.at runtime? According to the doc: https://man7.org/linux/man-pages/man2/clock_nanosleep.2.html it may return enotsup.

Reply:
According to this man indeed it is possible. However this change only makes the check stricter even it can't catch this tricky case but it wouldn't be caught before either.

Reply:
You were right, it's what was happening all along